### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by Victoria129**
- I have added styling to the application.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -6,20 +6,18 @@
 }
 
 body {
-  margin: 0;
+  display: flex;
+  align-items: center;
```
Comment (BenMukebo) [inline src/index.css]: - [ ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code to No repeating of the same rule by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```css
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- JS file
```js
<div className="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { NavLink } from 'react-router-dom';
 const Navbar = () => (
```
Comment (BenMukebo) [inline src/components/Navbar/Navbar.js:6]: - [ ] Great job so far 👏🏻 However the Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). kindly Add the missing profile icon on the right side of the navigation bar 👇🏻 

im1

File: src/index.css [hunk 3]
```diff
@@ -6,20 +6,18 @@
 }
 
 body {
-  margin: 0;
+  display: flex;
```
Comment (BenMukebo) [inline src/index.css]: - Great job so far 👏🏻 However the Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/).
- [ ] Make sure the main page is aligned in the center, you need to fix the margin-left and margin-right 👍🏻
- [ ] The footer should have the total width of his parent container, kindly address this and make sure  Book Title,  Author, and Category input match the original design and the Add Book as well look 👇🏻 

- Your Design:

!
im1


- Original Design:

image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Victoria129,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

00Mit


- The reason for this is that there is no `MIT.md` file in the root directory of this project. Kindly attach the file. 👍 it will be nice to add  this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL]  Please, add a meaningful  `Project description`. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how. 😉 💯 

- [OPTIONAL ] Kindly fix the broken link. I suggest you deploy correctly your app, This will improve your README file 😉 👍🏻 

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @Victoria129, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Readme file ✔️
- Project styling ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**